frontend/components/ui/blocknote/blocks/quiz-multiple-choice-block.tsx

```
@@ -0,0 +1,177 @@
+        props.editor.updateBlock(props.block, {
+          props: { options: [...options, newOption] as any },
+        });
+      };
+
+      const handleRemoveOption = (id: string) => {
+        if (options.length <= 2) return;
+        const updatedOptions = options.filter(
+          (opt: AnswerOption) => opt.id !== id,
+        );
+        props.editor.updateBlock(props.block, {
+          props: { options: updatedOptions as any },
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)